keccak-air/Cargo.toml

```diff
@@ -37,3 +37,6 @@ name = "prove_baby_bear_poseidon2"
 
 [[example]]
 name = "prove_goldilocks_keccak"
+
+[features]
+parallel = ["p3-maybe-rayon/parallel"]
```

Comment:
Is this needed? We can turn it on directly with `--features p3-maybe-rayon/parallel`, or `--features parallel` since they're in the same workspace (though they might be split off in the future)

Reply:
I was trying to `cargo run --example .. --features=parallel` but it didn't want to enable it for the dep because it wasn't a top level feature? Maybe there's some way I'm missing. If not, we should only need this on entry crates, not all libs

Reply:
I've been doing

    RUST_LOG=info cargo run --example prove_baby_bear_keccak --release --features parallel